assets/src/edit-story/app/media/utils/getResourceFromMedia3p.js

```diff
@@ -66,41 +66,34 @@ import createResource from './createResource';
  */
 function getUrls(m) {
   if (m.type.toLowerCase() === 'image') {
```

Comment:
The changes in this file seem unrelated to this PR

Reply:
Oops sorry, ended up in the wrong PR.